.github/workflows/swift.yml

```diff
@@ -0,0 +1,19 @@
+name: Swift
```

Comment:
I think this workflow could have a better name. This will run on PRs and on merges to master so it could be `PR build` or something

Reply:
Same goes for file name